app-interface/dashboards/grafana-dashboard-rhose-processing-errors.yaml

```diff
@@ -0,0 +1,431 @@
+apiVersion: v1
+data:
+  FleetManagerDashboard.json: |-
```

Comment:
`processing-errors.json`?

Reply:
@r00ta it would make sense to me, but I just copied the existing `app-interface` dashboards and I don't want to mess things up given that I don't know exactly how those works.

@msmagnanijr could you please check the `app-interface` dashboard I created?

Reply:
@kostola I think it's interesting to update this name to the @r00ta  suggestion. 

We can see our dashboards here: https://grafana.stage.devshift.net/dashboards/f/LzU3bp9nz/rhose

Reply:
renamed to `ProcessingErrorDashboard.json`